packages/storage-plugin/schematics/src/ng-generate/keys-migration.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+import { workspaceRoot } from '@nrwl/devkit';
+import { join } from 'path';
+import { createWorkspace } from '../_testing';
+
+describe('Storage Plugin Migration', () => {
+  const ngxsSchematicRunner = new SchematicTestRunner(
+    '@ngxs/storage-plugin/schematics',
+    join(workspaceRoot, 'packages/storage-plugin/schematics/collection.json')
+  );
+
+  const testSetup = async () => {
+    const appTree = await createWorkspace();
+    appTree.overwrite('package.json', `{"dependencies": {"@ngxs/store": "3.8.2"}}`);
+
+    return { appTree };
+  };
+
+  it('migrate empty forRoot', async () => {
+    const { appTree } = await testSetup();
+
+    const newContent = `
+    import { NgxsStoragePluginModule } from '@ngxs/storage-plugin';
+    import { NgModule } from '@angular/core';
+    import { BrowserModule } from '@angular/platform-browser';
+
+    import { AppRoutingModule } from './app-routing.module';
+    import { AppComponent } from './app.component';
+
+      @NgModule({
+        declarations: [
+          AppComponent
+        ],
+        imports: [
+          BrowserModule,
+          AppRoutingModule,
+          NgxsStoragePluginModule.forRoot()
+        ],
+        bootstrap: [AppComponent]
+      })
+      export class AppModule { }
+    `;
+
+    appTree.overwrite('/projects/foo/src/app/app.module.ts', newContent);
+
+    const tree = await ngxsSchematicRunner.runSchematic('ngxs-keys-migration', {}, appTree);
+
+    const contentUpdate = tree!.readContent('/projects/foo/src/app/app.module.ts');
+    expect(contentUpdate).toContain(`keys: '*'`);
```

Comment:
this should probably be better
```suggestion
    expect(contentUpdate).toMatchSnapshot();
```

Reply:
Yes indeed, that's much better. I did this change